### PR TITLE
delay functionality

### PR DESCRIPTION
**Pull request by kacperbiegajski**
Issue:
https://github.com/poseidon/wait-for-status-checks/issues/142

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ## Features
 
 * Wait for GitHub Check Runs on a Pull Request
-* Configure the poll `interval` and `timeout`
+* Configure the poll `delay`, `interval` and `timeout`
```
Comment (dghubble) [inline README.md:11]: nit: Throughout this PR, can you put `delay` after the existing `interval` and `timeout` options? I don't expect most users to need it

File: README.md [hunk 2]
```diff
@@ -54,14 +54,18 @@ jobs:
 
 The action knows to exclude its own check run (i.e. the GitHub Actions job that runs `wait-for-status-check`)
 
+When some of your checks trigger with a delay (e.g. workflow is using `concurrency` and has to wait for previous to be cancelled), then correctly setup `delay` parameter. The action will wait given amount of seconds to let your checks start.
```
Comment (dghubble) [inline README.md:57]: I think we could do without this line, delay should be pretty intuitive without mentioning your specific scenario

**Review by dghubble — COMMENTED** [review]
minor points

File: README.md (hunk 1 above)
Comment (kacperbiegajski) [reply README.md]: Done :)

File: README.md (hunk 2 above)
Comment (kacperbiegajski) [reply README.md]: Removed :)

**Comment by dghubble** [comment]
Thanks

**Comment by kacperbiegajski** [comment]
Hey @dghubble,  have you got any idea when new version will be released?

**Pull request merged**